projects/blueberry/index.js

```diff
@@ -1,6 +1,33 @@
 const { compoundExports2 } = require('../helper/compound')
+const { abi } = require('./abi.json');
+const sdk = require('@defillama/sdk');
+
+const BANK = '0x9b06eA9Fbc912845DF1302FE1641BEF9639009F7';
+
+async function tvl(timestamp, block, chainBlocks) {
+  let total = 0;
+  const nextPositionId = await sdk.api.abi.call({
+    target: BANK,
+    abi: abi['getNextPositionId'],
+    block
+  }).output;
+
+  for (let i = 0; i < nextPositionId; i++ ) {
+    const positionValue = await sdk.api.abi.call({
+      target: BANK,
+      abi: abi['getPositionValue'],
+      params: i,
+      block
+    }).output;
+    total += positionValue;
+  }
+
+  const totalTvl = total + compoundExports2({ comptroller: '0xfFadB0bbA4379dFAbFB20CA6823F6EC439429ec2'});  
```

Comment:
this line wont work, but how intertwined are lending part & earn part? so the protocol is changing from lending to leverage farming?

Reply:
Hi, the leverage is from the lending market. It's been always a leverage protocol but we just had the money market portion turned on first as part of the protocol roll-out.